src/System.Text.Json/src/System/Text/Json/Node/JsonObject.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.Collections.Generic;
+
+#pragma warning disable CS1591
```

Comment:
NIT: we are putting the pragmas in different places of the classes. It would be better if you pick a place that you like and make all files consistent :smile: